assets/js/player.js

```diff
@@ -112,6 +112,38 @@ function addCurrentTimeToURL(url) {
     return urlUsed;
 }
 
+/**
+ * Timer that updates the timestamp on all external links
+ */
+player.ready(function () {
+    // YouTube links
+
+    let elem_yt_watch = document.getElementById('link-yt-watch');
+    let elem_yt_embed = document.getElementById('link-yt-embed');
+
+    let base_url_yt_watch = elem_yt_watch.getAttribute('data-base-url');
+    let base_url_yt_embed = elem_yt_embed.getAttribute('data-base-url');
+
+    setTimeout(() => {
```

Comment:
I think you meant to use `setInterval` here. `setTimeout` just delays the execution of the code by the given delay. 

```suggestion
    setInterval(() => {
```